tests/components/__snapshots__/source_list.spec.js.snap

```diff
@@ -2,620 +2,412 @@
 
 exports[`SourceList can remove search 1`] = `
 <Column>
-  <SelectAll
-    height={40}
-    isAllSelected={false}
-    onClick={undefined}
-    renderer={[Function]}
-    selectAllMessage="Select All"
-    selectedIds={Array []}
-  />
-  <ItemsList
+  <SourceList
```

Comment:
This test no longer tests what it's expected to test

Reply:
After thinking, I changed source_list to have additional export - export source_list without HOC wrapping, so it gave me ability to provide more test cases for both wrapped and not wrapped component. Please tell what you think.